src/main/user-impl/java/com/mysql/cj/jdbc/ha/ca/plugins/Monitor.java

```diff
@@ -162,11 +166,8 @@ private HostInfo copy(HostInfo src, Map<String, String> props) {
   }
 
   private int findShortestIntervalMillis() {
```

Comment:
is int the better return type here? (Asking cause I don't have knowledge on that yet)

Reply:
Int as return type for the method is consistent with variables that hold all configuration parameters. I don't see a big difference in int and long in this particular case. Maybe long is more suitable for millis.